lib/modules/dosomething/dosomething_mbp/dosomething_mbp.module

```diff
@@ -104,6 +104,13 @@ function dosomething_mbp_get_transactional_payload($origin, $params = NULL) {
       'MEMBER_COUNT' => dosomething_user_get_member_count(TRUE),
     ),
   );
+
+  if (isset($params['transactionals'])) {
```

Comment:
just a refactor note, this whole block is pretty repetitive, might be easier to have an array of values and add them to the `$payload` if they are set